### PR TITLE
Update cn.m3u

### LINKED ISSUE #17093: Broken: sports-cctv5+/cctv-5（1080p）/cctv-16（1080p）
### Broken Link

https://iptv-org.github.io/iptv/categories/sports.m3u

### What happened to the stream?

Constantly interrupts/lagging

### Notes (optional)

https://iptv-org.github.io/iptv/categories/sports.m3u
cctv5+/cctv-5（1080p）/cctv-16（1080p）can not play

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by iotv0815**
fill empty tvg-id with channel name
in addition, fix dead channels
fixes #17093

### CODE REVIEW

**Review by Dum4G — CHANGES_REQUESTED** [review]
image
Please make sure tvg-id exist in database [iptv-org.github.io](https://iptv-org.github.io/)

**Comment by BellezaEmporium** [comment]
AFAIK there's no dashes in a tvg-id.

**Comment by BellezaEmporium** [comment]
image

File: streams/cn.m3u [hunk 1]
```diff
@@ -1025,27 +1025,27 @@ http://ott.js.chinamobile.com/PLTV/3/224/3221227393/index.m3u8
...
-#EXTINF:-1 tvg-id="",浙江教科影视 (1080p) [Not 24/7]
+#EXTINF:-1 tvg-id="ZheJiang-EducationMovie.cn",浙江教科影视 (1080p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel004/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangDigitalTimesChannel.cn",浙江数码时代 (1080p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel012/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江新闻
+#EXTINF:-1 tvg-id="ZheJiang-News.cn",浙江新闻
 https://ali-m-l.cztv.com/channels/lantian/channel007/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江民生
+#EXTINF:-1 tvg-id="ZheJiang-Life.cn",浙江民生
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1048]: Channel is the same than the one below.

Is it the "Minsheng Leisure" channel or the "Life" channel ? (last one being inexistant in the database).

File: streams/cn.m3u [hunk 2]
```diff
@@ -1025,27 +1025,27 @@ http://ott.js.chinamobile.com/PLTV/3/224/3221227393/index.m3u8
...
-#EXTINF:-1 tvg-id="",浙江少儿
+#EXTINF:-1 tvg-id="ZheJiang-Kids2.cn",浙江少儿
 https://ali-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel08/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江影视 (720p)
+#EXTINF:-1 tvg-id="ZheJiang-Movie2.cn",浙江影视 (720p)
 https://hw-m-l.cztv.com/channels/lantian/channel05/720p.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1039]: Doesn't exist anymore.

Either https://qiniup-m-l.cztv.com/channels/lantian/channel05/1080p.m3u8 (loop for now)
or https://ali-m-l.cztv.com/channels/lantian/channel05/1080p.m3u8 (loop for now)
or https://ct-m-l.cztv.com/channels/lantian/channel05/1080p.m3u8 (loop for now)

File: streams/cn.m3u [hunk 3]
```diff
@@ -1025,27 +1025,27 @@ http://ott.js.chinamobile.com/PLTV/3/224/3221227393/index.m3u8
...
+#EXTINF:-1 tvg-id="ZheJiang-Int-2.cn",浙江国际
 https://ali-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江国际 (1080p)
+#EXTINF:-1 tvg-id="ZheJiang-Int3.cn",浙江国际 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江少儿
+#EXTINF:-1 tvg-id="ZheJiang-Kids2.cn",浙江少儿
 https://ali-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1035]: ZhejiangEducationTechnologyChannel.cn per the image above.

Alt https://qiniup-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
or https://ct-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8

File: streams/cn.m3u [hunk 4]
```diff
@@ -1025,27 +1025,27 @@ http://ott.js.chinamobile.com/PLTV/3/224/3221227393/index.m3u8
 https://hw-m-l.cztv.com/channels/lantian/channel01/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangSatelliteTV.cn",浙江卫视 (576p)
 http://183.207.248.71/gitv/live1/G_ZHEJIANG/G_ZHEJIANG
-#EXTINF:-1 tvg-id="",浙江国际
+#EXTINF:-1 tvg-id="ZheJiang-Int-2.cn",浙江国际
 https://ali-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江国际 (1080p)
+#EXTINF:-1 tvg-id="ZheJiang-Int3.cn",浙江国际 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1031]: Alt https://qiniup-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
or https://ct-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8

image

File: streams/cn.m3u [hunk 5]
```diff
@@ -975,29 +975,29 @@ https://tmpstream.hyrtv.cn/hygg/sd/live.m3u8
...
+#EXTINF:-1 tvg-id="ZheJiang-Economic.cn",浙江 Ⅰ 浙江经济 (720p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel03/720p.m3u8
 #EXTINF:-1 tvg-id="",浙江 Ⅰ 绍兴影视 (720p)
 http://live.shaoxing.com.cn/video/s10001-sxtv3/index.m3u8
-#EXTINF:-1 tvg-id="",浙江 Ⅰ 绍兴综合 (576p)
+#EXTINF:-1 tvg-id="ZheJiang-General.cn",浙江 Ⅰ 绍兴综合 (576p)
 http://live.shaoxing.com.cn/video/s10001-sxtv1/index.m3u8
-#EXTINF:-1 tvg-id="",浙江公共 (1080p) [Not 24/7]
+#EXTINF:-1 tvg-id="ZheJiang-Public.cn",浙江公共 (1080p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel07/1080p.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1001]: Alt https://qiniup-m-l.cztv.com/channels/lantian/channel007/1080p.m3u8
or https://ct-m-l.cztv.com/channels/lantian/channel007/1080p.m3u8

image

File: streams/cn.m3u [hunk 6]
```diff
@@ -975,30 +975,32 @@ https://tmpstream.hyrtv.cn/hygg/sd/live.m3u8
...
 http://live.shaoxing.com.cn/video/s10001-sxtv3/index.m3u8
-#EXTINF:-1 tvg-id="",浙江 Ⅰ 绍兴综合 (576p)
+#EXTINF:-1 tvg-id="ZheJiangGeneral.cn",浙江 Ⅰ 绍兴综合 (576p)
 http://live.shaoxing.com.cn/video/s10001-sxtv1/index.m3u8
-#EXTINF:-1 tvg-id="",浙江公共 (1080p) [Not 24/7]
-https://hw-m-l.cztv.com/channels/lantian/channel07/1080p.m3u8
+#EXTINF:-1 tvg-id="ZheJiangPublic.cn",浙江公共 (1080p) [Not 24/7]
+https://qiniup-m-l.cztv.com/channels/lantian/channel007/1080p.m3u8
+#EXTINF:-1 tvg-id="ZheJiangPublic2.cn",浙江公共 (1080p) [Not 24/7]
+ https://ct-m-l.cztv.com/channels/lantian/channel007/1080p.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1003]:
```suggestion
https://ct-m-l.cztv.com/channels/lantian/channel007/1080p.m3u8
```

File: streams/cn.m3u [hunk 7]
```diff
@@ -1025,27 +1027,29 @@ http://ott.js.chinamobile.com/PLTV/3/224/3221227393/index.m3u8
...
-#EXTINF:-1 tvg-id="",浙江少儿
-https://ali-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
+#EXTINF:-1 tvg-id="ZheJiangInt3.cn",浙江国际 (1080p)
+https://qiniup-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
+#EXTINF:-1 tvg-id="ZheJiangInt4.cn",浙江国际 (1080p)
+https://ct-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
+#EXTINF:-1 tvg-id="ZheJiangKids2.cn",浙江少儿
+https://ct-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1039]:
```suggestion
https://qiniup-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
#EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p)
https://ct-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
```

**Comment by Dum4G** [comment]
Time has come? @BellezaEmporium

**Comment by BellezaEmporium** [comment]
There's still some work to do but you may approve what's already been done.

**Review by Dum4G — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**